### DISCUSSION TITLE
Request for Set Up documentation

### DISCUSSION THREAD

**Discussion by ColtonIdle** [GitLiveApp/firebase-kotlin-sdk#627] [Q&A]
I had to follow these docs in order to set this up: https://funkymuse.dev/posts/kmp-firebase/

Any chance we could get official docs. It took me an insanely long time to setup firebase kmp. Thank you!

**Comment by bpappin** [comment]
I agree, I'm stumbling around here wasting time, trying to figure out which part of the mixed up docs I need to be following for KMM.
I just sone some succinct setup documentation, so I dont need to spend days jumping all over the play and trying things.

**Comment by mylockerbiz** [comment]
the various places you can get some code examples may be years old and no longer accurate.
It is a total waste of time having to search around for some valid guidance.
**It should be coming from GitLiveApp staff.**
